src/lib.rs

```diff
@@ -50,7 +50,7 @@ pub mod clock;
 #[cfg(feature = "std")]
 pub mod ctap;
 #[cfg(not(feature = "std"))]
-mod ctap;
+pub mod ctap;
```

Comment:
Interesting. I'm surprised it used to compile before. I would have expected that the `Ctap::state()` doesn't compile because it is public and its return type is private.

The idea behind a private `ctap` module is that we consider it internal implementation detail. Users should use the `Ctap` struct.

Given that this is already broken, I'd suggest simply using `pub mod ctap` for now (without the `feature = "std"` switch). We could also remove the comment about testing given that it's outdated (although it's the intent).

Reply:
Yeah, I was a little confused by this also.

As suggested, I've remove the comment and the feature switch.

Reply:
This doesn't look resolved. Missing commit?

Reply:
Whoops, sorry about that.

Fixed.

Reply:
Maybe that's because `Ctap::state()` could have also been behind `#[cfg(feature = "std")]` ? (And we should add that.)

One possible solution to keep it private would be to move `ctap2::ctap::hid` to `ctap2::hid`. I had this option in mind when I last refactored the HID module, and there is not much left that stops us from doing so.

Reply:
Can I leave that refactor for you do to?

Reply:
Yeah why not, please add a TODO for me.

Reply:
Done.